x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/aws_credentials_form/get_aws_credentials_form_options.tsx

```diff
@@ -131,6 +135,7 @@ export const getAwsCredentialsFormOptions = (): AwsOptions => ({
         label: i18n.translate('xpack.csp.awsIntegration.roleArnLabel', {
           defaultMessage: 'Role ARN',
         }),
+        dataTestId: 'roleArnInput',
```

Comment:
let's move to a constant in `x-pack/plugins/cloud_security_posture/public/components/test_subjects.ts`

Reply:
and in other places where you have test subjects